src/views/CreateNew/DeployView.vue

```diff
@@ -295,9 +304,11 @@
       timer.value += 1
     }, 1000)
     executionId.value = route.params.id
+    applicationName.value = deployStore.getApplicationName
   })
 
   onUnmounted(() => {
     clearInterval(intervalRef.value)
+    deployStore.removeApplicationName()
```

Comment:
Creio que seria melhor remover o Applciation Name do store apenas quando o deploy for finalizado, sendo assim caso o usuário der refresh na pagina do deploy em andamento, ainda será possível visualizar o nome